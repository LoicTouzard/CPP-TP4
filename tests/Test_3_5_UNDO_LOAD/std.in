LOAD test3_5.txt
UNDO
LIST
EXIT
