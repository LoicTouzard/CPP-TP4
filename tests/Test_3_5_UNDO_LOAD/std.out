#1elements have been loaded successfully.
OK
OK
#The list is empty
